### PR TITLE
Several improvements

### PR DESCRIPTION
**Pull request by amanda-tarafa**

### CODE REVIEW

**Review by sethvargo — APPROVED** [review]

File: .kokoro-windows/New-BuildEnv.ps1 [hunk 1]
```diff
@@ -22,36 +22,41 @@ get-command choco -ErrorAction Stop
 choco list -li | Tee-Object -Variable chocoList
 $chocoPackages = ($chocoList) -join ' '
 
-if (-not $chocoPackages.Contains('Microsoft .NET Core SDK - 2.1.4')) {
-    choco install -y --sxs dotnetcore-sdk --version 2.2.203
+if (-not $chocoPackages.Contains('Microsoft .NET Core SDK - 2.2.')) {
```
Comment (jskeet) [inline .kokoro-windows/New-BuildEnv.ps1:25]: Not sure we should be using 2.2 at all - that went out of support even longer ago than 2.1.

Can we get rid of it entirely?

File: spanner/api/Spanner.Samples.Tests/GetDatabaseDdlAsyncTest.cs [hunk 2]
```diff
@@ -48,9 +48,9 @@ public async Task TestGetDatabaseDdlAsync()
         await _spannerFixture.RunWithTemporaryDatabaseAsync(_spannerFixture.InstanceIdWithMultiRegion, databaseId, async databaseId =>
         {
             var sample = new GetDatabaseDdlAsyncSample();
-            var statements =
+            var response =
                 await sample.GetDatabaseDdlAsync(_spannerFixture.ProjectId, _spannerFixture.InstanceIdWithMultiRegion, databaseId);
-            Assert.Collection(statements,
+            Assert.Collection(response.Statements,
```
Comment (jskeet) [inline spanner/api/Spanner.Samples.Tests/GetDatabaseDdlAsyncTest.cs:53]: I don't really understand how this change works - I'd have expected it to fail before. But hey, if it passes, that's fine.

File: spanner/api/Spanner.Samples.Tests/ListBackupOperationsTest.cs [hunk 3]
```diff
@@ -25,11 +24,12 @@ public ListBackupOperationsTest(SpannerFixture spannerFixture)
...
 
-    [Fact(Skip = "https://github.com/GoogleCloudPlatform/dotnet-docs-samples/issues/1583")]
+    [Fact]
     public void TestListBackupOperations()
     {
         ListBackupOperationsSample getBackupOperationsSample = new ListBackupOperationsSample();
-        var operations = getBackupOperationsSample.ListBackupOperations(_spannerFixture.ProjectId, _spannerFixture.InstanceId, _spannerFixture.BackupDatabaseId).ToList();
-        Assert.True(operations.Count >= 0);
+        var operations = getBackupOperationsSample.ListBackupOperations(_spannerFixture.ProjectId, _spannerFixture.InstanceId, _spannerFixture.BackupDatabaseId);
+        // We iterated over operation on the sample itself, let's not do that again.
```
Comment (jskeet) [inline spanner/api/Spanner.Samples.Tests/ListBackupOperationsTest.cs:32]: Is this actually testing anything useful then?

Comment (jskeet) [reply spanner/api/Spanner.Samples.Tests/ListBackupOperationsTest.cs:32]: (Same question for tests below.)

**Review by jskeet — APPROVED** [review]
A few questions, and I don't know enough about the Spanner tests to know whether the changes here reduce our confidence that the tests will catch problems, but otherwise fine.

File: .kokoro-windows/New-BuildEnv.ps1 (hunk 1 above)
Comment (amanda-tarafa) [reply .kokoro-windows/New-BuildEnv.ps1:25]: There are like 10 sample applications that are using 2.2. Those are all web application so moving them away from 2.2 takes time.
We have:

- #1548 
- #1004

File: spanner/api/Spanner.Samples.Tests/GetDatabaseDdlAsyncTest.cs (hunk 2 above)
Comment (amanda-tarafa) [reply spanner/api/Spanner.Samples.Tests/GetDatabaseDdlAsyncTest.cs:53]: The sample method was previously returning the statements only, I made it return the response object.

File: spanner/api/Spanner.Samples.Tests/ListBackupOperationsTest.cs (hunk 3 above)
Comment (amanda-tarafa) [reply spanner/api/Spanner.Samples.Tests/ListBackupOperationsTest.cs:32]: Just that the method is returning. I had it at Assert.NotEmpty but it still reached quota.

I could just call the method and have the test green if the method does not fail, but I'm hoping we can do better in the future so I left everything ready.

**Review by amanda-tarafa — COMMENTED** [review]
All questions answered.

**Pull request merged**